src/errors/custom-errors.ts

```diff
@@ -0,0 +1,7 @@
+export default class CustomError extends Error {
```

Comment:
Avoid using the word `Custom`, `My`, ... It's meaningless.

Instead, we can have 
```js
export default class NotFoundError extends Error {
  public statusCode: number;
  constructor(message: string) {
    super(message);
    this.statusCode = 404;
  }
}
```

```js
export default class BadRequestError extends Error {
  public statusCode: number;
  constructor(message: string) {
    super(message);
    this.statusCode = 400;
  }
}
```

... 

Or replace `Custom` by `Domain`